CRM/Contribute/Page/ManagePremiums.php

```diff
@@ -107,36 +107,35 @@ public function run() {
    */
   public function browse() {
     // get all custom groups sorted by weight
-    $premiums = [];
-    $dao = new CRM_Contribute_DAO_Product();
-    $dao->orderBy('name');
-    $dao->find();
-
-    while ($dao->fetch()) {
-      $premiums[$dao->id] = [];
-      CRM_Core_DAO::storeValues($dao, $premiums[$dao->id]);
-      // form all action links
-      $action = array_sum(array_keys($this->links()));
-
-      if ($dao->is_active) {
+    $premiums = \Civi\Api4\Product::get()
```

Comment:
I think we want permissions to be checked here, but I can get slightly confused at times in what context we do or don't want to pass in `FALSE` so this wants checking on review.